eng/native/functions.cmake

```diff
@@ -367,6 +367,25 @@ function(install_symbol_file symbol_file destination_path)
   endif()
 endfunction()
 
+function(install_static_library targetName destination component)
```

Comment:
this seems to be part of #62570 and not used anywhere in the build.

Reply:
This is used in all CMakeLists.txt files under nativeaot.